docs/arc42/01_introduction_and_goals.md

```diff
@@ -0,0 +1,74 @@
+This document follows the [arc42 template](https://arc42.org/).
+
+# Introduction and Goals
+
+This document describes LAPIS (Lightweight API for Sequences),
+which is a platform to give easy access to genomic sequence data alongside metadata of the sequenced probes.
+It is used to filter potentially large sequence data and return the result to the user through web access,
+so that a user can develop their own evaluation of the data.
+
+In the past we built https://genspectrum.org/ which involves such an API, but it is specialized for SARS-CoV-2.
+In this approach, we want to develop a generalized API that is configurable for a wider range of pathogens, especially
+viruses.
```

Comment:
```suggestion

The initial implementation of LAPIS is specialized for SARS-CoV-2 and is used by [CoV-Spectrum](https://cov-spectrum.org).
In this approach, we want to develop a generalized API that is configurable for a wider range of pathogens.
```

- Currently, I'm using "GenSpectrum" as the overall name for the organization and the whole project encompassing all dashboards, LAPIS, and other outputs.
- Striking "viruses" because maybe, it's easier than expected to include at least some bacteria instances.